index.d.ts

```diff
@@ -549,6 +543,18 @@ declare module 'discord.js' {
 		width: number;
 	}
 
+	export class MessageMentions {
+		channels: Collection<string, GuildChannel>;
```

Comment:
Perhaps have this as `Collection<string, TextChannel>` given that you cannot mention voice channels? Will save on typecasting for anyone working with channel mentions since it's the `TextChannel` they're going to want in the end.